### DISCUSSION TITLE
Bounding boxes (box2d) for images with semantic segmentation

### DISCUSSION THREAD

**Discussion by NemesisIrae** [bdd100k/bdd100k#208] [Q&A] [answered]
Hello. 
I wanted to train an image detector on the images which have semantic segmentation available. The detector operates on ground truth in the form of bounding boxes. I can't seem to find box2d annotation anywhere in the segmentation data set. I've searched the 100k data set and located only 3426 images that are also present in the semantic segmentation set (out of 8000 available images - 7000 training and 1000 validation).
Is really just that small portion of the semantic segmentation dataset also annotated with box2d?

Thanks in advance :)

**Comment by thomasehuang** [comment, marked as answer]
You can check the instance segmentation labels, which includes segmentation masks for each object in the segmentation data set. From that, you can compute the bounding boxes from the instance masks.

**Comment by iremonur** [comment]
Hi @thomasehuang, I want to train a detector with the BDD100K data in which traffic cones are present. In the dataset, the traffic cones were annotated with segmentation masks, therefore I plan to compute traffic cone BBOX'es from these segmentation masks. However, a corresponding image is not available for each segmentation mask. Where could I find the corresponding image to each segmentation mask?

**Comment by thomasehuang** [comment]
Each mask should correspond to an image in the 10K subset, documented [here](https://doc.bdd100k.com/download.html#id1). The filenames should match.
